### ISSUE TITLE
Version 6.1 Style Transfer still producing garbage out. 

### ISSUE THREAD

**Issue by rainhut** [apple/turicreate#3018]
Starting with version 6.0, the Style Transfer is no longer working. This was issue was reported in version 6.0, so I tried again with 6.1 and got the same result.  

1. I'm using the default settings. 
2. I have a MacBook Pro 16 inch with a GPU card. 
3. The python script is killed at 34,000 "iterations. zsh: killed     python classifier.py"
4. So I set max_iterations = 33000 and ran again and tried the mlmodel. 
5. The result is the same as in version 6.0. The images that use the created mlmodel come out as garbage. See the image link below for an example. I'm also attaching a link to the original image and a zip file with the generated mlmodel.
6. FYI, I've built 12 mlmodels with versions prior to 6.0 that are ok. 

![turicreate6-1sample](https://user-images.githubusercontent.com/1415094/75113154-6906c580-5608-11ea-803d-debac670b945.jpg)

![originalimage](https://user-images.githubusercontent.com/1415094/75113229-2c879980-5609-11ea-8d2e-a5a7aed636b3.jpg)

[planets13.mlmodel.zip](https://github.com/apple/turicreate/files/4241339/planets13.mlmodel.zip)

Rob

**Comment by rainhut** [comment]
I think the problem is using a GPU on Mac.  Conflicting docs. Rebuilding with CPU. I’ll report back in two days.

**Comment by TobyRoseman** [comment]
Yes, please let us know the results of your investigation regarding using the CPU rather than the GPU.

Also what are the "conflicting docs" you mention?

**Comment by rainhut** [comment]
With CPU only training, I'm not getting the garbage out. So 6.1 does not yet support GPU on a Mac. Unfortunately, CPU is a lot slower.

The docs conflict is that the 5.8 release notes say under New Features that "Style Transfer supports training with Mac GPU.

But the Style Transfer documentation has not changed and it still says that GPU is not supported. 

The attached image uses a model built with CPU. I used the same parameters, style and content images as the one from the GPU training. 

![styletransfer6-1](https://user-images.githubusercontent.com/1415094/75222494-931fcb00-5761-11ea-8a4f-6bdffe49dee5.jpg)

**Comment by saddif** [comment]
Hi -i am getting the same issue with AMD GPU as above - Macbook pro 2018 AMD 560x using turicreate 6.2, 6.1 and 6.0. 
None of these releases seem to use the Nvidia GPU on Colabs either - they automatically train using CPU only however if i switch to  Turicreate 5.8 and install mxnet-cu100==1.4.0.post0 the Nvidia GPU is picked up and training works.

Is the AMD GPU support a known issue, when do you think it will be fixed?

@rainhut your CPU output looks very good - please can you share your parameter settings, i have not been able to get anything as cool as that! 

Thanks
